test/game_test.rb

```diff
@@ -1,9 +1,7 @@
-require 'SimpleCOV'
```

Comment:
Since we need this at some point, maybe we need to figure out how to run rake without SimpleCov interfering?

Reply:
Agreed, I think we can make a card for this on our project board so it doesn't get overlooked